config/ConnectorConfig.json

```diff
@@ -9,7 +9,8 @@
    },
    "AppConfig":{
       "CycleTimeMs":1000,
-      "PostDuplicateTagValues": false
+       "PostDuplicateTagValues": false,
+       "OMF":false

```

Comment:
just noticing the space after the : on the postduplicatevalue line, that should probably get removed in its own whitespace commit.